### PR TITLE
feat: rsync off1 from ovh3 (until we move to full zfs sync)

### PR DESCRIPTION
**Pull request by alexgarel**
fixes:
- #186

### CODE REVIEW

File: scripts/ovh3/rsync_off1.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+nice -n 19 ionice -c 3 \
+  rsync -a -x -e "ssh -T -x" \
+  --exclude="lost+found" --exclude '*.old' \
+  off@off1.openfoodfacts.org:/srv/ $backup_dir/srv/  &
+# sync of /srv2
+for dir_name in off off-pro
+do
+  nice -n 19 ionice -c 3 \
+    rsync -a -x -e "ssh -T -x" \
+    off@off1.openfoodfacts.org:/srv2/$dir_name $backup_dir/srv2/$dir_name --exclude=html/images/prod
```
Comment (stephanegigandet) [inline scripts/ovh3/rsync_off1.sh:17]: a copy paste introduced an extra line feed I think

File: scripts/ovh3/rsync_off1.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ucts/ --exclude=orgs/ --exclude=users/ &
```
Comment (stephanegigandet) [inline scripts/ovh3/rsync_off1.sh:18]: there's no orgs and users dirs in /srv2/*/

Comment (alexgarel) [reply scripts/ovh3/rsync_off1.sh:18]: it's in srv2/off :-) but it should be skipped the same (because those are symlinks to zfs datasets)

**Comment by stephanegigandet** [comment]
I'm guessing this is not needed anymore

**Comment by alexgarel** [comment]
This is not needed anymore.

**Pull request closed without merging**